javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/OpenTelemetryInstaller.java

```diff
@@ -65,6 +67,8 @@ public static synchronized void installAgentTracer(Config config) {
             });
       }
 
+      InstrumentationContext.internalSetContextStoreSupplier(
+          (keyClass, contextClass) -> FieldBackedProvider.getContextStore(keyClass, contextClass));
```

Comment:
I'd probably put that somewhere in `InstrumentationLoader` where `InstrumentationContextProvider` implementation gets chosen. Maybe that `getContextStore()` method should be a part of that provider interface?

Reply:
Moved to `InstrumentationModuleInstaller`